samples/sample_common/src/vaapi_utils.cpp

```diff
@@ -400,6 +400,8 @@ VAStatus CLibVA::AcquireVASurface(
     ctx = (AcquireCtx*)calloc(1, sizeof(AcquireCtx));
     if (!ctx) return VA_STATUS_ERROR_OPERATION_FAILED;
 
+    ctx->fd = -1;
```

Comment:
I am trying to search where this `fd` is actually used and I actually don't see usages. Is it actually used somewhere? If not - we can just delete the code entirely.

Reply:
> I am trying to search where this `fd` is actually used and I actually don't see usages. Is it actually used somewhere? If not - we can just delete the code entirely.
both the modifications can resolve the issues, the fd exists may because it have  other reasons. Shall we have the patch in to so that we can fix current issue.

Reply:
> I am trying to search where this `fd` is actually used and I actually don't see usages. Is it actually used somewhere? If not - we can just delete the code entirely.
both the modifications can resolve the issues, the fd exists may because it have  other reasons. Shall we have the patch in to so that we can fix current issue.

Reply:
OK, how about keeping AcquireCtx as wrapper and remove the member of fd, in case new member added in future?

Reply:
I am fine with that. Still can't believe `fd` is not actually used... Let's see if build will pass.

Reply:
so, build is ok.